server/chat/backend/agent/tools/gitlab_tool.py

```
@@ -0,0 +1,576 @@
+                (suggestion_id, user_id),
+            )
+            row = cursor.fetchone()
+            if row:
+                suggestion = {
+                    "id": row[0], "incident_id": str(row[1]), "title": row[2],
+                    "description": row[3], "file_path": row[4], "original_content": row[5],
+                    "suggested_content": row[6], "user_edited_content": row[7],
+                    "repository": row[8], "commit_message": row[9],
+                    "pr_url": row[10], "created_branch": row[11],
+                }
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)